Dockerfile

```diff
@@ -80,10 +83,15 @@ ENV PIP_SRC=/deps/src/
 ENV NPM_CONFIG_PREFIX=/deps/
 ENV SWIG_FEATURES="-D__x86_64__"
 
+RUN useradd -Md /code/ olympia
```

Comment:
Possibly a bit counter-intuitive to set `$HOME` to `/code` even if we did this historically -  does anything actually rely on that behaviour?

Reply:
I don't think so and I'm not sure we actually need that. I could try without and see where that gets us